From: Matthias Brugger <mbrugger@suse.com>
Subject: x86: boot: rename reserve_bios_regions
Patch-mainline: Never, this fixes a compilation issue
References: bsc#1031345

Commit 
edce21216a88 ("x86/boot: Reorganize and clean up the BIOS area reservation code")
renames reserve_ebda_region() to reserve_bios_regions(). But we don't have this 
commit in our tree. Nevertheless we have commit
4971531af319 ("x86/efi: Test for EFI_MEMMAP functionality when iterating EFI memmap")
which calls reserve_bios_regions().

This patch renames reserve_bios_regions() back to reserve_ebda_region() to fix 
the compilation issue

Singed-off-by: Matthias Brugger <mbrugger@suse.com>
---
 arch/x86/kernel/setup.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/x86/kernel/setup.c
+++ b/arch/x86/kernel/setup.c
@@ -1106,7 +1106,7 @@ void __init setup_arch(char **cmdline_p)
 	memblock_set_current_limit(ISA_END_ADDRESS);
 	memblock_x86_fill();
 
-	reserve_bios_regions();
+	reserve_ebda_region();
 
 	if (efi_enabled(EFI_MEMMAP)) {
 		efi_fake_memmap();
